carmm/build/slab_consistent_bulk_generator.py

```diff
@@ -0,0 +1,106 @@
+
+# This functionality will help to generate a bulk model from a slab.
+
+from ase import Atoms
+
+
+def is_close_to_integer(arr, tolerance=1e-5):
```

Comment:
Is this supposed to be a private routine to help the work flow? I might make this private with _ at the front, and also move down the file (as lower priority than the function below it)